TAScheduler/AcceptanceTests/AssignTAtoLab tests.py

```diff
@@ -0,0 +1,120 @@
+import datetime
+
+from django.test import TestCase
+
+from TAScheduler.models import User, TA, Course, TAToCourse
+
+
+#
+class SuccessfulAssignment(TestCase):
+    user = None
+    ta = None
+    course = None
+    tatocourse = None
+
+    def setUp(self):
+        self.user = User.objects.create(email_address="ta@class.com", password="password", first_name="first",
+                                        last_name="last", home_address="there", phone_number=1234567890)
+
+        self.ta = TA.objects.create(user=self.user)
+
+        self.course = Course.objects.create(course_id=100, semester="fall 2023", name="test",
+                                            description="test", num_of_sections=2, modality="online", credits=1)
+
+        self.tatocourse = TAToCourse.objects.create(ta=self.ta, course=self.course)
+
+    def test_creation(self):
+        temp = TAToCourse.objects.create(ta=self.ta, course=self.course)
+        self.assertIsNotNone(temp, "TAToCourse does not make anything")
+        self.assertEqual(temp.course, self.course, "TAToCourse does not make correct course")
+        self.assertEqual(temp.ta, self.ta, "TAToCourse does not make correct TA")
+
+
+class NoTA(TestCase):
+    user = None
+    ta = None
+    course = None
+    tatocourse = None
+
+    def setUp(self):
+        self.user = User.objects.create(email_address="ta@class.com", password="password", first_name="first",
+                                        last_name="last", home_address="there", phone_number=1234567890)
+
+        self.ta = TA.objects.create(user=self.user)
+
+        self.course = Course.objects.create(course_id=100, semester="fall 2023", name="test",
+                                            description="test", num_of_sections=2, modality="online", credits=1)
+
+        self.tatocourse = TAToCourse.objects.create(ta=self.ta, course=self.course)
+
+    def test_no_ta(self):
+        temp = TAToCourse.objects.create(ta=self.ta)
+        self.assertIsNone(temp.ta, "TAToCourse makes TA with no input")
+
+
+class NoCourse(TestCase):
+    user = None
+    ta = None
+    course = None
+    tatocourse = None
+
+    def setUp(self):
+        self.user = User.objects.create(email_address="ta@class.com", password="password", first_name="first",
+                                        last_name="last", home_address="there", phone_number=1234567890)
+
+        self.ta = TA.objects.create(user=self.user)
+
+        self.course = Course.objects.create(course_id=100, semester="fall 2023", name="test",
+                                            description="test", num_of_sections=2, modality="online", credits=1)
+
+        self.tatocourse = TAToCourse.objects.create(ta=self.ta, course=self.course)
+
+    def test_no_course(self):
+        temp = TAToCourse.objects.create(course=self.course)
+        self.assertIsNone(temp.course, "TAToCourse makes course with no input")
+
+
+class TaDelete(TestCase):
```

Comment:
I have labeled your AC in Trello to better communicate what each AC is supposed to do. Each class in this file needs to be dedicated to a specific AC. ATM, there are no AC for PBI #12 dedicated to handling #TADelete

HOWEVER,
I was thinking about modifying the design documents sometime tomorrow to include more features ... this could be 1 features
(Thank you for the idea!)